dotcom-rendering/src/web/components/Picture.tsx

```diff
@@ -1,320 +1,320 @@
+					{ breakpoint: breakpoints.desktop, width: 640 },
+					{ breakpoint: breakpoints.leftCol, width: 800 },
+					{ breakpoint: breakpoints.wide, width: 880 },
+				];
+			case 'supporting':
+				return [
+					{ breakpoint: breakpoints.mobile, width: 445 },
+					{ breakpoint: breakpoints.mobileLandscape, width: 605 },
+					{ breakpoint: breakpoints.phablet, width: 620 }, // tablet is also 620px
+					{ breakpoint: breakpoints.desktop, width: 300 }, // leftCol is also 300px
+					{ breakpoint: breakpoints.wide, width: 380 },
+				];
+			case 'thumbnail':
+				return [
+					{ breakpoint: breakpoints.mobile, width: 120 }, // mobileLandscape and tablet are also 120px
+					{ breakpoint: breakpoints.tablet, width: 140 }, // desktop, leftCol and wide are also 140px
+				];
+			case 'immersive':
+				return [
+					{ breakpoint: breakpoints.mobile, width: 465 },
+					{ breakpoint: breakpoints.mobileLandscape, width: 645 },
+					{ breakpoint: breakpoints.phablet, width: 725 },
+					{ breakpoint: breakpoints.tablet, width: 965 },
+					{ breakpoint: breakpoints.desktop, width: 1125 },
+					{ breakpoint: breakpoints.leftCol, width: 1140 },
+					{ breakpoint: breakpoints.wide, width: 1300 },
+				];
+			case 'halfWidth':
+				return [{ breakpoint: breakpoints.mobile, width: 445 }];
+			case 'inline':
+			default:
+				return [
+					{ breakpoint: breakpoints.mobile, width: 445 },
+					{ breakpoint: breakpoints.mobileLandscape, width: 605 },
+					{ breakpoint: breakpoints.phablet, width: 620 },
+				];
+		}
 	}
-
-	return desiredWidths;
 };
 
 /**
- * Returns the desired width for an image at the specified breakpoint, based on the image role, format, and if it's main media.
- * This function acts as a source of truth for widths of images based on the provided parameters
+ * Generates a url for calling the Fastly Image Optimiser
  *
- * Returns 'breakpoint', or 'breakpoint / x' when the image is expected to fill a % of the viewport,
- * instead of a fixed width.
+ * @see {@link https://developer.fastly.com/reference/io/}
  *
- * @param breakpoint
- * @param role
- * @param isMainMedia
- * @param format
- * @returns {Pixel}
  */
-const getDesiredWidthForBreakpoint = (
-	breakpoint: number,
-	role: RoleType,
-	isMainMedia: boolean,
-	format: ArticleFormat,
-): Pixel => {
-	if (
-		(format.display === ArticleDisplay.Showcase ||
-			format.display === ArticleDisplay.NumberedList) &&
-		isMainMedia
-	) {
-		// Showcase main media images (which includes numbered list articles) appear
-		// larger than in body showcase images so we use a different set of image sizes
-		if (breakpoint >= breakpoints.wide) return 1020;
-		if (breakpoint >= breakpoints.leftCol) return 940;
-		if (breakpoint >= breakpoints.tablet) return 700;
-		if (breakpoint >= breakpoints.phablet) return 700;
-		return breakpoint;
-	}
+const generateSignedUrl = ({
+	master,
+	imageWidth,
+	quality,
+}: {
+	master: string;
+	imageWidth: number;
+	quality: 'low' | 'high';
```

Comment:
I'm not sure the word `quality` is adequate, here. I would prefer `resolution` or `density`, as higher resolution images get a lower quality value.